src/clj/game/cards/agendas.clj

```diff
@@ -1163,6 +1164,8 @@
             {:event :runner-turn-ends
              :req (req (seq (filter #(= (:zone %) [:servers zone :ices])
                                     (all-active-installed state :corp))))
+             :duration :end-of-turn
+             :unregister-once-resolved true
```

Comment:
Only needed if it's expected to fire and before the duration happens. Given that this is a conditional ability, it'll be cleared at turn end.